config/database.yml

```diff
@@ -0,0 +1,25 @@
+# SQLite. Versions 3.8.0 and up are supported.
+#   gem install sqlite3
+#
+#   Ensure the SQLite 3 gem is defined in your Gemfile
+#   gem "sqlite3"
+#
+default: &default
+  adapter: sqlite3
+  pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
+  timeout: 5000
```

Comment:
- [ ] Looking at `config/database.yml`, we can see that we've not met the following project requirement:
  
  > - Make sure that your project has Postgres database set up.
  
  The project is currently set up with `sqlite3`. Please set it up with Postgres. This [GUIDE](https://www.geeksforgeeks.org/how-to-implement-postgresql-database-in-rails-application/#:~:text=So%20to%20add%20PostgreSQL%20database,~%3E%201.4'%20from%20gemfile.&text=After%20adding%20pg%20gem%20in%20gemfile%20run%20given%20command%20on%20terminal.&text=This%20command%20installs%20all%20necessary%20files%20for%20implement%20the%20PostgreSQL%20database.) might help.